plugins/pa11y/readme.md

```diff
@@ -0,0 +1,54 @@
+# @dotcom-tool-kit/pa11y
+ A Tool Kit plugin to run [Pa11y](https://github.com/pa11y/pa11y). This plugin runs accessibility tests on your pages via the command line or Node.js, so you can automate your testing process. Pa11y runs automatically on the `test:review`hook, using the information about the app from the state.
+
+ ## Installation and usage
+   With Tool Kit [already set up](https://github.com/financial-times/dotcom-tool-kit#installing-and-using-tool-kit), install this plugin as a dev dependency:
+
+```
+    npm install --save-dev @dotcom-tool-kit/pa11y
+```
+
+And add it to your repo's `.toolkitrc.yml`:
+
+```
+plugins:
+    - '@dotcom-tool-kit/pa11y'
+```
+
+## Pa11y configurations added on the project
+Pa11ySchema:
+
+| Value | Type | Description |
+|:---:|:---:|---|
+| host | string | Domain against pa11y will run |
+| wait | number | The time in milliseconds to wait before running HTML CodeSniffer on the page |
+| exceptions | array.string | An array of result codes and types that you'd like to ignore. You can find the codes for each rule in the console output and the types are error, warning, and notice. Note: warning and notice messages are ignored by default |
+| hideElements | array.string | A CSS selector to hide elements from testing, selectors can be comma separated. Elements matching this selector will be hidden from testing by styling them with visibility: hidden |
```

Comment:
I was about to approve it, but then saw this - 

**suggestion**: hideElements is `string` instead of `array.string`